settings.gradle

```diff
@@ -9,7 +9,7 @@ if (localPropsFile.exists()) {
     throw new IllegalStateException("local.properties could not be located for build process")
 }
 
-if (gradle.ext.BUILD_CORE_FROM_SOURCE.toBoolean()) {
+if (gradle.ext.hasProperty('BUILD_CORE_FROM_SOURCE') && gradle.ext.BUILD_CORE_FROM_SOURCE.toBoolean()) {
```

Comment:
Seems gradle.ext.has is synonym for gradle.hasProperty

Reply:
OK